src/backend/controllers/LikeController.js

```diff
@@ -45,17 +46,32 @@ export const addItemToLikedVideos = function (schema, request) {
   const user = requiresAuth.call(this, request);
   if (user) {
     const { video } = JSON.parse(request.requestBody);
-    if (user.likes.some((item) => item.id === video.id)) {
-      return new Response(
-        409,
-        {},
-        {
-          errors: ['The video is already in your liked videos'],
-        }
-      );
+    // check if video present in db
+    const foundVideo = this.db.videos.findBy({ _id: video._id });
+    if (foundVideo) {
+      if (user.likes.some((item) => item._id === video._id)) {
+        return new Response(
+          409,
+          {},
+          {
+            errors: ['The video is already in your liked videos'],
+          }
+        );
+      }
+
+      user.likes.push(video);
+      // Updating Like in Videos DB
+      this.db.videos.update(foundVideo, { likes: foundVideo.likes + 1 });
```

Comment:
add `try catch` block

Reply:
try-catch isn't there in Mockbee, So, I will skip it for now. Otherwise, It will require a whole refactor of all files.